recipes/types-JACK-Client/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "types-JACK-Client" %}
+{% set version = "0.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/types-JACK-Client-{{ version }}.tar.gz
+  sha256: 0a9524b20d361f2adec9a646f4c9e217c89acb3795463b24b1394acb46f9962c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - test -f $SP_DIR/jack-stubs/__init__.pyi
+
+
+about:
+  home: https://github.com/python/typeshed
+  summary: Typing stubs for JACK-Client
+  license: Apache-2.0
```

According to the license, this should be
```suggestion
  license: Apache-2.0 AND MIT
```